samples/taxonomy/infrastructure/units.yaml

```diff
@@ -0,0 +1,14 @@
+definitions:
+  Units:
```

Comment:
Do we want all the different types of units in one enum?  Or should there be separate ones for cost, size, etc?

Reply:
Will be handled in https://github.com/fybrik/fybrik/issues/1331